src/backends/cudanative/cudanative.jl

```diff
@@ -146,7 +146,7 @@ immutable CUFunction{T}
     kernel::T
 end
 
-if success(`nvcc --version`)
+if try success(`nvcc --version`); catch false; end
```

Comment:
`nvcc` is properly detected by CUDArt, so why not use it from there?
Also, CUDArt doesn't build without it, so it should be unconditionally available (if CUDArt has been built).

Reply:
Ah good to know! I will need to rewrite this part for your compilation PR anyways.